app/main.py

```diff
@@ -1,3 +1,41 @@
-def shop_trip():
-    # write your code here
-    pass
+import json
+from app.customer import Customer
+from app.car import Car
+from app.shop import Shop
+
+
+def shop_trip() -> None:
+
+    with open("app/config.json", "r") as config:
+        config_dict = json.load(config)
+        fuel_cost = config_dict["FUEL_PRICE"]
```

Comment:
don't write this and belllow code in `with open("app/config.json", "r") as config:`